### PR TITLE
Prettify SQL queries

### PR DESCRIPTION
**Pull request by jonatanschroeder**
Custom package and rules to prettify SQL queries. The formatted queries are implemented in a separate PR to ensure this one can include the other one in git-blame-ignore. Sprocs are not included as sql-formatter has some issues with them.

### CODE REVIEW

File: .git-blame-ignore-revs
Comment (jonatanschroeder) [inline .git-blame-ignore-revs:13]: TODO: After #7152 is merged, replace with the hash of that PR.

**Review by mwest1066 — APPROVED** [review]

File: package.json
Comment (jonatanschroeder) [reply package.json:151]: I was actually waiting for your review on this to confirm that this is fine, since I noticed it's different than the other package reference. I didn't do anything fancy here, only `yarn add`, and it set the version as listed. I'll change it to `workspace:*`.

Comment (jonatanschroeder) [reply package.json]: Done in 8567fb516.

**Pull request merged**
